packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -3073,7 +3073,15 @@ const schema: SchemaType<"Posts"> = {
     label: "stale-while-revalidate caching enabled",
     group: formGroups.adminOptions,
     ...schemaDefaultValue(false),
-  }
+  },
+  generateDraftJargon: {
+    type: Boolean,
+    optional: true,
+    hidden: true,
+    ...schemaDefaultValue(false),
```

Comment:
for readability's sake I'd prefer to have object spreads at the end, unless you're explicitly overriding one of the properties from the spread by assigning a new value to that field after the spread